src/index.d.ts

```diff
@@ -0,0 +1,2 @@
+type Expand<T> = T extends infer O ? { [K in keyof O]: O[K] } : never;
```

Comment:
Whoa conditional typing 🤯

Reply:
It's a good one! This lets the type hover of these props that are mostly pass-throughs to antd be easier to navigate. Ripped from stack overflow, probably lol